cmake/modules/SearchInstalledSoftware.cmake

```diff
@@ -1865,3 +1865,31 @@ if(NOT ROOT_HAVE_CXX_ATOMICS_WITHOUT_LIB)
     set(ROOT_ATOMIC_LIBS ${ROOT_ATOMIC_LIB})
   endif()
 endif()
+
+#------------------------------------------------------------------------------------
+# Check if the pyspark package is installed on the system.
+# The FindPySpark.cmake module will first look for pyspark dependencies:
+# 1. It will look for Java in the system. Minimum version 1.8
+# 2. It will try to `import py4j` using PYTHON_EXECUTABLE_Development_Main
+#
+# If the two depencies are met, the module will try to `import pyspark` using the same
+# Python executable. If everything could be found then the PYSPARK_FOUND variable will
```

Comment:
Maybe "is usable" instead of spelling out the (current) dependencies of pyspark?
```suggestion
# If `pyspark` can be imported and is usable then the PYSPARK_FOUND variable will
```